### ISSUE TITLE
Adding code to call all methods

### ISSUE THREAD

**Issue by antgonza** [biocore/mds-approximations#1]
The methods are:
- [x] eigh (@hannesholste)
- [x] scmds (@mortonjt)
- [x] nystrom (@mortonjt)
- [x] svd (@hannesholste)
- [x] ssvd (@hannesholste)
- [x] fsvd (@hannesholste)
- [x] eigsh (@hannesholste)

**Comment by rob-knight** [comment]
While we’re looking at these, it might be worth seeing if there are parallel libraries for svd that could be called direct from ipython as this is a hard limit on what we can do currently? I cc:ed Fernando in case he knows of any that are drop-in and highly parallel.

**Comment by mortonjt** [comment]
Note that [eigh](http://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.linalg.eigh.html), [eigsh](http://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.linalg.eigsh.html) and [svd](http://docs.scipy.org/doc/scipy-0.16.0/reference/generated/scipy.linalg.svd.html) are already available in scipy.  

[ssvd](http://scikit-learn.org/stable/modules/generated/sklearn.decomposition.TruncatedSVD.html) is also implemented in scikit-learn

**Comment by wasade** [comment]
Highly recommend adding in scikit-learn to the mix. Decomposition methods [here](http://scikit-learn.org/stable/modules/decomposition.html#decompositions)

**Comment by wasade** [comment]
btw, there was work done within scikit-bio on this. I _think_the present branch is this [one](https://github.com/wasade/scikit-bio/tree/PCoA_to_pcoa)

**Comment by wasade** [comment]
@rob-knight, it is likely that numpy and scipy are already highly parallel here. We typically do not link against OpenBLAS but it is natively parallel and numpy/scipy will take advantage of it if linked. That was one of subtle but potentially important things we learned at DS4DS

**Comment by ElDeveloper** [comment]
Looks like @mortonjt, has already tackled my share of this, issue, what method should I focus on then?
